<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFImFCSSetParamFromFileDialog</class>
 <widget class="QDialog" name="QFImFCSSetParamFromFileDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>imFCS: set parameter from RDR/file/...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,0">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,1">
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>parameter to update:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cmbParameter"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout" columnstretch="1,0">
         <property name="topMargin">
          <number>5</number>
         </property>
         <item row="0" column="0">
          <widget class="QRadioButton" name="radRDR">
           <property name="text">
            <string>from other fit result</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QWidget" name="verticalWidget" native="true">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="btnLoadFile">
              <property name="text">
               <string>load from file</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="radOtherSource">
           <property name="text">
            <string>from other source</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="radOverview">
           <property name="text">
            <string>from overview</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" rowspan="2">
          <widget class="QWidget" name="widget" native="true">
           <layout class="QFormLayout" name="formLayout">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>RDR:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QFRDRComboBox" name="cmbRDR"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>result group:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QFResultsGroupComboBox" name="cmbResultGroup"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>parameter:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QFRawDataResultsComboBox" name="cmbResult"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="labInfo">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkMapDesired">
         <property name="text">
          <string>show image mapped to desired</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFPlotter" name="pltData" native="true">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget1">
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
       <item>
        <widget class="QCheckBox" name="chkImageTransform">
         <property name="text">
          <string>image transform</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFImageTransformWidget" name="imageTransfrom" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnHelp">
       <property name="text">
        <string>&amp;Help</string>
       </property>
       <property name="icon">
        <iconset resource="../../lib/lib.qrc">
         <normaloff>:/lib/help/help.png</normaloff>:/lib/help/help.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFPlotter</class>
   <extends>QWidget</extends>
   <header>qfplotter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QFRDRComboBox</class>
   <extends>QComboBox</extends>
   <header>qfrdrcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QFRawDataResultsComboBox</class>
   <extends>QComboBox</extends>
   <header>qfrawdataresultscombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QFResultsGroupComboBox</class>
   <extends>QComboBox</extends>
   <header>qfresultsgroupcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QFImageTransformWidget</class>
   <extends>QWidget</extends>
   <header>qfimagetransform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbRDR</tabstop>
  <tabstop>cmbResultGroup</tabstop>
  <tabstop>cmbResult</tabstop>
 </tabstops>
 <resources>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QFImFCSSetParamFromFileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QFImFCSSetParamFromFileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radOtherSource</sender>
   <signal>toggled(bool)</signal>
   <receiver>verticalWidget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>654</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radRDR</sender>
   <signal>toggled(bool)</signal>
   <receiver>QFImFCSSetParamFromFileDialog</receiver>
   <slot>updateDataFromRDR()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radRDR</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbRDR</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>QFImFCSSetParamFromFileDialog</receiver>
   <slot>updateDataFromRDR()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>453</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbResultGroup</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>QFImFCSSetParamFromFileDialog</receiver>
   <slot>updateDataFromRDR()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>453</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbResult</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>QFImFCSSetParamFromFileDialog</receiver>
   <slot>updateDataFromRDR()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>453</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>484</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkImageTransform</sender>
   <signal>toggled(bool)</signal>
   <receiver>imageTransfrom</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>903</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>667</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radOverview</sender>
   <signal>toggled(bool)</signal>
   <receiver>QFImFCSSetParamFromFileDialog</receiver>
   <slot>updateDataFromRDR()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>553</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>538</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkImageTransform</sender>
   <signal>toggled(bool)</signal>
   <receiver>QFImFCSSetParamFromFileDialog</receiver>
   <slot>replotOvr()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>736</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>909</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkMapDesired</sender>
   <signal>toggled(bool)</signal>
   <receiver>QFImFCSSetParamFromFileDialog</receiver>
   <slot>replotOvr()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>replotOvr()</slot>
  <slot>updateDataFromRDR()</slot>
 </slots>
</ui>
